app/src/main/res/layout/activity_token.xml

```diff
@@ -1,59 +1,122 @@
-    tools:context="crux.bphc.cms.activities.TokenActivity">
+    android:layout_height="match_parent">
 
-    <ImageView
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/token_constraint"
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:contentDescription="@null"
-        android:elevation="-1dp"
-        android:scaleType="centerCrop"
-        android:src="@drawable/intro_bg"
-        app:tint="?tokenImageTint" />
-
-    <ImageView
-        android:id="@+id/logo"
-        android:layout_width="200dp"
-        android:layout_height="200dp"
-        android:layout_marginBottom="64dp"
-        android:contentDescription="@null"
-        android:scaleType="fitXY"
-        app:srcCompat="@drawable/ic_bits_logo"
-
-        app:layout_constraintBottom_toTopOf="@+id/google_login"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintHorizontal_bias="0.5"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintVertical_chainStyle="packed" />
-
-    <com.google.android.gms.common.SignInButton
-        android:id="@+id/google_login"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        app:buttonSize="wide"
-        app:colorScheme="auto"
-
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintHorizontal_bias="0.5"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/logo" />
-
-    <TextView
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_margin="16dp"
-        android:text="@string/powered_by_crux"
-        android:textColor="#fff"
-        android:textSize="18sp"
-        android:textStyle="bold"
-
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintEnd_toEndOf="parent" />
-
-</androidx.constraintlayout.widget.ConstraintLayout>
+        android:layout_height="match_parent">
+
+        <ImageView
+            android:id="@+id/imageView4"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:contentDescription="@null"
+            android:elevation="-1dp"
+            android:scaleType="centerCrop"
+            android:src="@drawable/intro_bg"
+            app:layout_anchor="@+id/token_constraint"
+            app:layout_anchorGravity="center"
+            app:tint="?tokenImageTint" />
+
+        <ImageView
+            android:id="@+id/bits_logo_image"
+            android:layout_width="150dp"
+            android:layout_height="150dp"
+            android:layout_marginTop="16dp"
+            android:layout_marginBottom="8dp"
+            android:contentDescription="@string/bits_pilani_logo"
```

Comment:
Set this to `@null`. This image is purely decorative and doesn't need to be read by a talkback reader.

Reply:
Remove `bits_pilani_logo` in string.xml as well.